framework/src/media/audio/audio_manager.c

```diff
@@ -48,6 +48,10 @@
 #define AUDIO_DEVICE_OUT_SPEAKER		0x0200
 #define AUDIO_DEVICE_OUT_HEADSET		0x0400
 
+#define DEFAULT_AUDIO_DEVICE_PATH		"/dev/audio/"
```

Comment:
There's config related this. please check Kconfig in audio

Reply:
I searched all Kconfgs but couldn't find the config...